.github/workflows/main.yml

```diff
@@ -0,0 +1,25 @@
+name: Run tests
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
```

Comment:
use `make docker_tests` so that the tests run with psql

Reply:
Although I'll cc @annabauza here. Do we want to tests to run on sqlite or on postgre? I think postgre makes sense because it would be closer to the production env

Reply:
Actually please just run it on postgres because we're using postgres in production

Reply:
yeah I if it's possible the would be great to run tests on postgress. Thank you

Reply:
Should I run it with docker??

Reply:
@anshsrtv  it's ok like this. just increase python version below. Thanks